Dockerfile

```diff
@@ -0,0 +1,87 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ARG PY_VER=3.6.9
+FROM python:${PY_VER} AS superset-py
+
+RUN mkdir /app \
+        && apt-get update -y \
+        && apt-get install -y --no-install-recommends \
+            build-essential \
+            default-libmysqlclient-dev \
+            libpq-dev \
+        && rm -rf /var/lib/apt/lists/*
+
+COPY requirements* ./setup.py ./MANIFEST.in ./README.md ./app/
+COPY superset /app/superset
+
+RUN cd /app \
+        && pip install --upgrade setuptools pip \
+        && pip install -r requirements.txt -r requirements-dev.txt \
```

Comment:
I understand the need, but I really don't like this `requirements-dev` here, contains lot's of things that are not needed and his changed by contributions taking into context a development environment not an official docker build. We could use a `requirements-extra` for this.